media/inf/syssetup.inf.tpl

```diff
@@ -1,15 +1,12 @@
-;
```

Comment:
Please revert.

Reply:
see below? What is it purpose? Many other inf files don't have such.